config/routes.rb

```diff
@@ -10,4 +10,7 @@
 
   root "home#index"
   resources :organizations
-end
+
+  get '/attachments/:id', to: 'attachments#show', as: 'attachment'
+  delete '/attachments/:id', to: 'attachments#destroy'
```

Comment:
we can combine these routes like this: `resources :attachments, only: [:show, :destroy]`